teams/blau/beschleunigung.m

```diff
@@ -160,9 +167,11 @@ function setupNodeGrid()
                 %update costs for neighbours
                 movementCostToNeighbour = currentNode.gCost + norm(currentNode.worldPos - neighbour.worldPos);
                 if (movementCostToNeighbour < neighbour.gCost || ~containsNode(openSet, neighbour.gridPos))
+                    
+                    mineCost = nodeGrid(neighbour.gridPos(1), neighbour.gridPos(2)).mineCost;
                     nodeGrid(neighbour.gridPos(1), neighbour.gridPos(2)).gCost = movementCostToNeighbour;
                     nodeGrid(neighbour.gridPos(1), neighbour.gridPos(2)).hCost = norm(endp - neighbour.worldPos);
-                    nodeGrid(neighbour.gridPos(1), neighbour.gridPos(2)).fCost = neighbour.gCost + neighbour.hCost;
+                    nodeGrid(neighbour.gridPos(1), neighbour.gridPos(2)).fCost = neighbour.gCost + neighbour.hCost + mineCost;
                     nodeGrid(neighbour.gridPos(1), neighbour.gridPos(2)).parent = currentNode.gridPos;
 
```

Comment:
hier wird fCost überschrieben! daher wird die Kostenberechnung für die Minen in costMine gespeichert und hier wieder addiert